pycbc/inference/sampler_emcee.py

```diff
@@ -233,8 +247,12 @@ def write_results(self, fp, max_iterations=None):
             large enough to accomodate future data.
         """
         self.write_metadata(fp)
-        self.write_chain(fp, max_iterations=max_iterations)
-        self.write_likelihood_stats(fp, max_iterations=max_iterations)
+        self.write_chain(fp, start_iteration=start_iteration,
+                         end_iteration=end_iteration,
+                         max_iterations=max_iterations)
+        self.write_likelihood_stats(fp, start_iteration=start_iteration,
+                                    end_iteration=end_iteration,
+                                    max_iterations=max_iterations)
         self.write_acceptance_fraction(fp)
```

Comment:
It looks like you implemented the slicing for ``write_acceptance_fraction`` in kombine smapler but not emcee. Any reason for that?